src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,171 @@
+}
+
+interface DayOverviewPdfProps {
+    data: DayOverviewData;
+}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
+    color: string;
+    fill: boolean;
+}
+
+const styles = StyleSheet.create({
+    page: {
+        display: "flex",
+        flexDirection: "column",
+        padding: "0 30pt 15pt",
+        fontSize: "11pt",
+    },
+    title: {
+        fontFamily: "Helvetica-Bold",
+        textTransform: "uppercase",
+        fontSize: "30px",
+        textAlign: "center",
+        margin: "0 22.5pt 22.5pt",
+    },
+    subtitleWrap: {
+        justifyContent: "space-between",
+        marginBottom: "4pt",
+    },
+    svg: {
+        margin: "0 0.75pt",
+        padding: "0 0.75pt",
+    },
+    cell: {
+        borderStyle: "solid",
+        borderRightWidth: 1,
+        borderBottomWidth: 1,
+        padding: "2.25pt",
+    },
+    margin: { height: "37.5pt" },
+    row: { display: "flex", flexDirection: "row" },
+    bold: { fontFamily: "Helvetica-Bold" },
+
+    cellLogo: { flex: 1 },
+    cellName: { flex: 4 },
+    cellPostcode: { flex: 4 },
+    cellTime: { flex: 2 },
+    cellCollection: { flex: 4 },
+    cellInstructions: { flex: 10 },
+});
+
+const dateTimeToAMPM = (datetime: string): string => {
+    return new Date(datetime).getHours() <= 11 ? "AM" : "PM";
+};
+
+// IconDefinition[4] is defined as string | string[] but in reality there is no use case
+// with string[] as svgPathData
+const CustomSVG: React.FC<CustomSVGProps> = ({ icon, color, fill }) => (
+    <Svg width={11} height={11} viewBox="0 0 512 512" style={styles.svg}>
+        <Path
+            fill={fill ? color : undefined}
+            stroke={color}
+            strokeWidth={40}
+            d={icon.icon[4] as string}
+        />
+    </Svg>
+);
+
+const DayOverviewMargin: React.FC<{}> = () => {
+    return <View style={styles.margin} fixed></View>;
+};
+
+const DayOverviewSubtitle: React.FC<DayOverviewSubtitleProps> = ({ subtitleKey, children }) => (
+    <Text>
+        <Text style={styles.bold}>{subtitleKey}: </Text>
+        {children}
+    </Text>
+);
+
+const DayOverviewHeader: React.FC<{}> = () => {
+    return (
+        <View style={[styles.row, styles.bold, { borderTop: "1 solid black" }]} fixed>
+            <Text style={[styles.cellLogo, styles.cell]}></Text>
+            <Text style={[styles.cellName, styles.cell]}>Name</Text>
+            <Text style={[styles.cellPostcode, styles.cell]}>Postcode</Text>
+            <Text style={[styles.cellTime, styles.cell]}>Time</Text>
+            <Text style={[styles.cellInstructions, styles.cell]}>Instructions</Text>
+        </View>
+    );
+};
+
+const DayOverviewRow: React.FC<DayOverviewRowProps> = ({ row }) => {
+    return (
+        <View style={styles.row} wrap={false}>
+            <View style={[styles.cellLogo, styles.cell, styles.row]}>
+                <CustomSVG icon={faSquare} color="black" fill={false} />
+                {row.clients!.flagged_for_attention && (
```

Comment:
will there be a case when `clients` is null?

Reply:
I think the database by default need a parcel to contain a foreign client key and if a client is deleted, then the parcels will be removed by cascade too (?) 
I have to double check with Tom if the foreign client key can be null though but I don't think so?

Reply:
yeah no that makes sense :)